src/index.html

```diff
@@ -11,7 +11,155 @@
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+            How old are You?
+          <input
+            type="number"
+            required
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-label">
+          Full date of birth:
+          <input type="date" required>
+        </label>
+
+        <label class="form-label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="registration" class="form-field">
+        <legend>Registration</legend>
+
+        <label class="form-label">
+          E-mail:
+          <input type="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Password:
+          <input
+            type="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="anInterestingFactAboutYou" class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-label">
+          Do you love cats?
+          <label>
+            <input
+              name="catLover"
+              type="radio"
+              value="yes"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              name="catLover"
+              type="radio"
+              value="no"
+            >
+              No
+          </label>
+        </div>
+
+
+        <label class="form-label">
+          What is your favorite color?
+          <input type="color">
+        </label>
+
+        <label class="form-label">
+            What time do you go to bed?
+          <input type="time">
+        </label>
+
+        <label class="form-label">
+          What are your favorite brand of cars?
+          <select name="favoriteCar" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-label">
+          How do you rate our work?
+          <input
+            type="range"
+            value="0"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="additionalInfo" class="form-field">
+        <legend>Additional info:</legend>
+
+        <label class="form-label">
+          Comments:
+          <textarea
+          name="feedback"
+          cols="20"
+          rows="2"
+          autocomplete="off"
+         ></textarea>
```

Comment:
attributes indentation
make sure to scan the rest of the code to comply with it
```suggestion
          <textarea
            name="feedback"
            cols="20"
            rows="2"
            autocomplete="off"
          ></textarea>
```